src/scss/_tables.scss

```diff
@@ -0,0 +1,166 @@
+$table-bg-color: #f2f2f2;
+$table-header-bg-color: rgba(#dcdcdc, 0.6);
+
+.table-container {
+    overflow-x: auto;
+    display: flex;
+    margin: 0.5em 5px 1em 5px;
+    width: auto;
+    border-radius: $element-border-radius;
+    @include card-shadow;
```

Comment:
I don't think we should have a shadow on the table. In this case, the table and the pagination controls should be on the same level yet the table is "above" the pagination controls. I think we should use shadows sparingly